### ISSUE TITLE
Test for WCAG success criterion1.4.5: Images of Text

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#210]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-text-presentation.html
Images of Text: If the technologies being used can achieve the visual presentation, text is used to convey information rather than images of text except for the following:
- Customizable: The image of text can be visually customized to the user's requirements;
- Essential: A particular presentation of text is essential to the information being conveyed.
## Applicable
- Images used
- Special fonts used

In practice: always
## Images of text not allowed
- [ ] Styled Headings - Correct: use CSS to present headings in a specific font and size.
- [ ] Navigation items - A Web page contains a menu of navigation links that have both an icon and text to describe their target. Correct: CSS is used to display the text's font family, size and foreground and background colors; as well as the spacing between the navigation links.
- [ ]  A quote -  A Web page contains a quote. Correct: The quote itself is presented as italicized text, indented from the left margin. The name of the person to whom the quote is attributed is below the quote with 1.5x the line space and further indented from the left margin. CSS is used to position the text; set the spacing between lines; as well as display the text's font family, size, color and decoration.
## Image of text allowed
- [ ] Dynamically Generated Images - A Web page uses server-side scripting to present text as an an image. The page includes controls that allow the user to adjust the font size and foreground and background colors of the generated image.
- [ ] A logo containing text -  A Web site contains the organization's logo in the top left corner of each Web page. The logo contains logotype (text as part, or all, of the logo). The visual presentation of the text is essential to the identity of the logo and is included as a gif image which does not allow the text characteristics to be changed. The image has a text alternative.
- [ ]  Representation of a font family - A Web page contains information about a particular font family. Substituting the font family with another font would defeat the purpose of the representation. The representation is included as a jpeg image which does not allow the text characteristics to be changed. The image has a text alternative.
- [ ] A representation of a letter - A Web page contains a representation of an original letter. The depiction of the letter in its original format is essential to information being conveyed about the time period in which it was written. The letter is included as a gif image which does not allow the text characteristics to be changed. The image has a text alternative.
- [ ] Symbolic text characters - A form allows users to enter blocks of text. The form provides a number of buttons, including functions to style the text and check spelling. Some of the buttons use text characters that do not form a sequence that expresses something in human language. For example "B" to increase font weight, "I" to italicize the text and "ABC" to check the spelling. The symbolic text characters are included as gif images which do not allow the text characteristics to be changed. The buttons have text alternatives.
- [ ]  Customizable font settings in images of text - A Web site allows users to specify font settings and all images of text on the site are then provided based on those settings. 
## Techniques
- [ ] C22: Using CSS to control visual presentation of text (CSS)
- [ ] C30: Using CSS to replace text with images of text and providing user interface controls to switch
- [ ] G140: Separating information and structure from presentation to enable different presentations 
## Relationships and QUAIL
- There is no failure described, but we could check for the defined items: navigation items, styled headings and quotes.
